wasi/Makefile

```diff
@@ -1,4 +1,5 @@
-CFLAGS += -std=c89 -Wunused-result -Wall -Wpedantic -Wno-long-long -Wno-unused-function
+# TODO Find out why limits.h doesn't define PATH_MAX on Manjaro Linux
```

Comment:
As far as I understand, `PATH_MAX` is not defined for `-std=c89`

Reply:
Ah yes.  That would explain it.

Reply:
Can you please remove this TODO? I'll try to address this in a follow-up PR